include/uzfs_io.h

```diff
@@ -35,4 +39,13 @@ extern int uzfs_read_data(void *zv, char *buf, uint64_t offset, uint64_t len,
 
 extern void uzfs_flush_data(void *zv);
 
+/*
+ * API to set/unset rebuilding mode
+ * If, rebuilding mode is set, then every normal write IO will be added to
+ * condensed avl tree (incoming io tree). For IO with is_rebuild
+ * flag set, it will be checked with incoming_io_tree and only non-overlapping
+ * part from IO will be written.
+ */
+extern void uzfs_set_rebuilding_mode(void *zv);
```

Comment:
pass the value either TRUE or FALSE to set, and have a get function

Reply:
ok